clients/wallet_daemon_client/src/types.rs

```diff
@@ -124,6 +129,16 @@ pub struct TransactionGetResponse {
     pub transaction_failure: Option<RejectReason>,
 }
 
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub struct TransactionGetAllResponse {
```

Comment:
Can we rename it to `GetAllTransactionsResponse` or to `GetAllTransactionsByStatusResponse` ?

Reply:
Same :-)